zenstruck/messenger-monitor-bundle/0.5/src/Entity/ProcessedMessage.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Entity;
+
+use Zenstruck\Messenger\Monitor\History\Model\ProcessedMessage as BaseProcessedMessage;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(readOnly: true)]
+#[ORM\Table('processed_messages')]
```

Comment:
perhaps add a prefix like the default `messenger_` table generation with doctrine transport ?
current name feel generic